meemoo/helpers.py

```diff
@@ -50,12 +58,11 @@ def try_to_find_md5(object_metadata):
     if re.match("^[a-fA-F0-9]{32}$", md5):
         return md5
     else:
-        raise KeyError(f"md5 not found or semantically incorrect. Found: '{md5}'")
+        raise InvalidEventException(f"md5 not found or semantically incorrect. Found: '{md5}'")
 
 
 def get_from_event(event, name):
```

Comment:
It would be better if the event gets parsed once, into an instance of an event class. Out of scope for this ticket.